cmd/solo-kit-gen/main.go

```diff
@@ -57,9 +51,6 @@ func run() error {
 	}
 
 	for _, inDir := range projectDirs {
-		outDir := strings.Replace(inDir, "api", "pkg/api", -1)
-
-		imports := append(commonImports, "-I"+inDir)
 
 		tmpFile, err := ioutil.TempFile("", "solo-kit-gen-")
 		if err != nil {
```

Comment:
I think something is messed up with the spacing below this line (not caused by this PR)

Reply:
Nvm, just looks weird in github